src/cfd/constants/constants.tsx

```diff
@@ -250,3 +251,20 @@ export const AppToIconMapper: Record<string, ComponentType<SVGAttributes<SVGElem
     huawei: InstallationHuaweiIcon,
     ios: InstallationAppleIcon,
 };
+
+export const getWebtraderUrl = ({ details }: { details: THooks.MT5AccountsList }) => {
+    return `${details?.white_label_links?.webtrader_url}?login=${details?.display_login}&server=${details?.server_info?.environment}`;
+};
+
+export const getDeeplinkUrl = ({ details }: { details: THooks.MT5AccountsList }) => {
+    return `metatrader5://account?login=${details?.display_login}&server=${details?.server_info?.environment}`;
+};
+
+export const getMobileAppInstallerUrl = ({ details }: { details: THooks.MT5AccountsList }) => {
+    if (mobileOsDetect() === 'iOS') {
```

Comment:
do we have any shared constant for these different OS ?

Reply:
nope @amina-deriv , this snippet is taken from legacy code and in legacy code there is no constant for the OS

Reply:
we can use constants for the OS here